guides/operations/setup/installing_sqream_with_monit.rst

```diff
@@ -103,10 +105,559 @@ Note the following:
+         $ #stop program = "/usr/bin/systemctl stop sqream5"
+         $ #SQREAM5-END
+         $ #       alert user@domain.com on {nonexist, timeout}
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+         $ #
+         $ #SQREAM6-START
+         $ #check process sqream6 with pidfile /var/run/sqream6.pid
+         $ #start program = "/usr/bin/systemctl start sqream6"
+         $ #stop program = "/usr/bin/systemctl stop sqream6"
+         $ #SQREAM6-END
+         $ #       alert user@domain.com on {nonexist, timeout}
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+         $ #
+         $ #SQREAM7-START
+         $ #check process sqream7 with pidfile /var/run/sqream7.pid
+         $ #start program = "/usr/bin/systemctl start sqream7"
+         $ #stop program = "/usr/bin/systemctl stop sqream7"
+         $ #SQREAM7-END
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+         $ #
+         $ #SQREAM8-START
+         $ #check process sqream8 with pidfile /var/run/sqream8.pid
+         $ #start program = "/usr/bin/systemctl start sqream8"
+         $ #stop program = "/usr/bin/systemctl stop sqream8"
+         $ #SQREAM8-END
+         $ #       alert user@domain.com on {nonexist, timeout}
+         $ #                      with mail-format {
+         $ #                            from:     Monit@$HOST
+         $ #                            subject:  sqream2 $EVENT - $ACTION
+         $ #                            message:  This is an automate mail, sent from monit.
+         $ #                    }
+
+Starting Monit
+====================================  
+
+**To start Monit:**
+
+1. Stop all actively running SQream services:
+
+   .. code-block:: console
+     
+      $ sudo systemctl stop sqream[1-4]  #This command stops sqream1, sqream2,..,sqream4
+      $ sudo systemctl stop serverpicker
+      $ sudo systemctl stop metadataserver
+
+2. Start Monit as a super user:
+
+   .. code-block:: console
+     
+      $ sudo systemctl start monit
+   
+3. Verify that the following SQream processes are running and listening:
+  
+   .. code-block:: console
+     
+      $ sudo systemctl status metadataserver
```

Comment:
I'd make a comment that this and the following lines are applicable on the main server

Reply:
That they are **only** applicable on the main server?

Reply:
Yes, only on the main server